Framework/MDAlgorithms/test/MergeMDTest.h

```diff
@@ -126,6 +126,27 @@ class MergeMDTest : public CxxTest::TestSuite {
     // Remove workspace from the data service.
     AnalysisDataService::Instance().remove(outWSName);
   }
+
+  void test_displayNormalization() {
+    // Name of the output workspace.
+    std::string outWSName("MergeMDTest_OutputWS");
+    auto ws0 =
+        AnalysisDataService::Instance().retrieveWS<IMDEventWorkspace>("ws0");
+    ws0->setDisplayNormalization(API::MDNormalization::NoNormalization);
+    ws0->setDisplayNormalizationHisto(
+        API::MDNormalization::NumEventsNormalization);
+    auto ws = execute_merge(outWSName);
+    if (!ws)
+      return;
+
```

Comment:
If `ws` is null the test should fail. I'll create a separate issue for you to address this.